doc/macsec-gearbox/PAI-Proposal-Failover.md

```diff
@@ -0,0 +1,254 @@
+# External PHY Failover
+
+# Proposal
+
+| **Title** | **External PHY Abstraction Interface** |
+| --- | --- |
+| **Authors** | **Broadcom** |
+| **Status** | **In review** |
+| **Type** | **Standards track** |
+| **Created** | **8/31/2020** |
+| **SAI-Version** | **0.1** |
+
+
+# **Contents**
+
+[List of Changes](#_Toc52272605)
+
+[1 Overview](#_Toc52272606)
+
+[1.1 Failover Configuration on PHY](#_Toc52272607)
+
+[1.2 Additions to saiport.h](#_Toc52272608)
+
+[1.3 Additions to saiswitch.h](#_Toc52272609)
+
+[2 Configuration Example](#_Toc52272610)
+
+# Overview
+
+The purpose of this document is to describe PHY failover (FO) functionality and interface to manage failover functionality of the PHY. PHY is the connector between MAC (SerDes) and physical medium such as optical fiber or copper transceivers. PHY failover feature uses two physical port configurations: primary and secondary. This can be enabled on system/host or line/client side of a port. It provides the ability to switch from primary port to secondary port or vice versa without losing data/link.
+
+![](figures/phy_failover_on_host_side.png)
```

Comment:
the location is not correct. you can put figures in macsec-gearbox directory and reference from file directly using the file name.